wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift

```diff
@@ -398,6 +411,8 @@ final class ConversationListViewController: UIViewController {
         searchController.searchBar.isTranslucent = false
         searchController.searchResultsUpdater = self
 
+        searchController.searchBar.placeholder = searchPlaceholderText
```

Comment:
It would be nice for this to be tested. One way to achieve this without too much change is to extract this logic into the following at test that without the need to instantiate the SUT. Of course you could also do snapshot tests I imagine.

```swift
private static func makeSearchController(filter: ConversationFilter?) -> UISearchController {
  // ...
}
```

Reply:
Done!